### ISSUE TITLE
String matching based on finite automaton: Grafik gibt falsche Prozentzahl zurück bzw berechnet falsch

### ISSUE THREAD

**Issue by Dudedism** [hydranox/foo-pub#136]
Erst mal ein Lob wegen den motivierenden Grafiken der eigenen Ergebnisse.

Als ich allerdings die oben 2te.String Matching Aufgabe bearbeitet habe, wurde die falsche Bewertung zurückgegeben. 
Die einzigen Fehler die ich hatte waren 2 Werte in der Tabelle. Kann es sein, dass bei einem Fehler in der Tabelle, die Bewertung auf 0 gesetzt wird?

Kein schlimmes Problem, allerdings ein wenig verwirrend... 
#schönen Tag noch

**Comment by alexanderj1988** [comment]
ja, das ist der String aus der DB, ob man die ganze Übung bestanden hat, nicht der Teilaufgabe.
Muss mir an der Stelle eine Alternative Überlegen.

mfg Alex
